From f5eeeb110c2e8344e8b097aaa159cf187ecc0124 Mon Sep 17 00:00:00 2001
From: Michel Alexandre Salim <salimma@fedoraproject.org>
Date: Thu, 14 Jul 2011 13:32:35 +0200
Subject: [PATCH] Fix desktop file for validation

- Add categories: TV depends on Video; P2P depends on Network
- MimeType listing should be semicolon-terminated
---
 tv/linux/miro.desktop |    4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/tv/linux/miro.desktop b/tv/linux/miro.desktop
index 6d3580a..e03f896 100644
--- a/tv/linux/miro.desktop
+++ b/tv/linux/miro.desktop
@@ -11,5 +11,5 @@ Icon=miro
 TryExec=miro
 Exec=miro %F
 Terminal=false
-Categories=GTK;AudioVideo;TV;Player;P2P;News;FileTransfer;
-MimeType=application/x-miro;application/x-democracy;application/x-bittorrent;video/dv;video/mpeg;video/x-mpeg;video/msvideo;video/quicktime;video/x-anim;video/x-avi;video/x-ms-asf;video/x-ms-wmv;video/x-msvideo;video/x-nsv;video/x-flc;video/x-fli;application/ogg;application/x-ogg;application/x-matroska;audio/x-mp3;audio/x-mpeg;audio/mpeg;audio/x-wav;audio/x-mpegurl;audio/x-scpls;audio/x-m4a;audio/x-ms-asf;audio/x-ms-asx;audio/x-ms-wax;application/vnd.rn-realmedia;audio/x-real-audio;audio/x-pn-realaudio;misc/ultravox;audio/vnd.rn-realaudio;audio/x-pn-aiff;audio/x-pn-au;audio/x-pn-wav;audio/x-pn-windows-acm;image/vnd.rn-realpix;video/vnd.rn-realvideo;audio/x-pn-realaudio-plugin;video/ogg;audio/ogg
+Categories=GTK;AudioVideo;Video;TV;Player;Network;P2P;News;FileTransfer;
+MimeType=application/x-miro;application/x-democracy;application/x-bittorrent;video/dv;video/mpeg;video/x-mpeg;video/msvideo;video/quicktime;video/x-anim;video/x-avi;video/x-ms-asf;video/x-ms-wmv;video/x-msvideo;video/x-nsv;video/x-flc;video/x-fli;application/ogg;application/x-ogg;application/x-matroska;audio/x-mp3;audio/x-mpeg;audio/mpeg;audio/x-wav;audio/x-mpegurl;audio/x-scpls;audio/x-m4a;audio/x-ms-asf;audio/x-ms-asx;audio/x-ms-wax;application/vnd.rn-realmedia;audio/x-real-audio;audio/x-pn-realaudio;misc/ultravox;audio/vnd.rn-realaudio;audio/x-pn-aiff;audio/x-pn-au;audio/x-pn-wav;audio/x-pn-windows-acm;image/vnd.rn-realpix;video/vnd.rn-realvideo;audio/x-pn-realaudio-plugin;video/ogg;audio/ogg;
-- 
1.7.6

